server/utils/comments/comments.util.js

```diff
@@ -16,6 +17,29 @@ class commentValidation {
     }
     return next();
   }
+
+  /**
+   * Check User of article
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Object} next
+   * @returns {Object} response
+   */
+  static async checkUser(req, res, next) {
+    const { id } = req.params;
+    const userId = req.user.id;
+
+    const userComment = await Comment.findOne({
+      where: {
+        userId,
+        id
+      }
+    });
+    if (!userComment) {
+      return Response(res, 400, 'Only the user of this comment can edit article');
```

Comment:
Nice one @BukkyOmo. Kindly fix the typo here.

Reply:
It's been fixed. Thanks